src/actions/photosActions.ts

```diff
@@ -435,72 +456,25 @@ export function fetchPhotoDetail(image_hash) {
   };
 }
 
-export function simpleFetchPhotos() {
-  return function (dispatch) {
-    dispatch({ type: "FETCH_PHOTOS" });
-    Server.get("photos/", { timeout: 100000 })
-      .then((response) => {
-        dispatch({
-          type: "FETCH_PHOTOS_FULFILLED",
-          payload: response.data.results,
-        });
-      })
-      .catch((err) => {
-        dispatch({ type: "FETCH_PHOTOS_REJECTED", payload: err });
-      });
-  };
-}
-
-export function fetchTimestampPhotos() {
-  return function (dispatch) {
-    dispatch({ type: FETCH_PHOTOSET });
-    Server.get("albums/date/photohash/list/", { timeout: 100000 })
-      .then((response) => {
-        var photosGroupedByDate = response.data.results;
-        adjustDateFormat(photosGroupedByDate);
-        dispatch({
-          type: FETCH_PHOTOSET_FULFILLED,
-          payload: {
-            photosGroupedByDate: photosGroupedByDate,
-            photosFlat: getPhotosFlatFromGroupedByDate(photosGroupedByDate),
-            photosetType: PhotosetType.TIMESTAMP,
-          },
-        });
-      })
-      .catch(getFetchPhotosetErrorHandler(dispatch));
-  };
-}
-
-export function fetchNoTimestampPhotoList() {
-  return function (dispatch) {
-    dispatch({ type: FETCH_PHOTOSET });
-    Server.get("photos/notimestamp/list/", { timeout: 100000 })
-      .then((response) => {
-        var photosFlat = response.data.results;
-        dispatch({
-          type: FETCH_PHOTOSET_FULFILLED,
-          payload: {
-            photosFlat: photosFlat,
-            photosetType: PhotosetType.NO_TIMESTAMP,
-          },
-        });
-      })
-      .catch(getFetchPhotosetErrorHandler(dispatch));
-  };
-}
-
+const _PaginatedPigPhotosSchema = z.object({
+  count: z.number(),
+  next: z.number().nullable(),
```

Comment:
previous and next are strings not numbers. It returns the whole path and not just the next/previous page number. 
And I just noticed, that my count API is not needed, as it is also returned by the paginated query 😅

Reply:
Fixed.

I didn't do anything about the count request in this PR. I think this PR servers to highlight these types of things. There is still room for improvement. :D